src/pages/SwapV2/index.tsx

```diff
@@ -947,6 +958,41 @@ export default function Swap() {
                         </GreyCard>
                       ) : showApproveFlow ? (
                         <>
+                          <Card mb="24px" backgroundColor={rgba(theme.subText, 0.2)} padding="12px">
```

Comment:
move it into a new component to prevent conflict vs HungDoan's pr

Reply:
Done.